tests/functional_tests/test_accessibility.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+from flask import url_for
+import re
+from playwright.sync_api import Page
+from axe_core_python.sync_playwright import Axe
+import json
+import os
+
+ACCESSIBILITY_STANDARDS = ["wcag2a", "wcag2aa"]
+
+def check_accessibility(page: Page):
+    '''
+    Inserts axe core python into a page at the yield step
+    to run accessibility based testing. Axe will run on
+    the page defined in the function.
+    '''
+    if page.title() != "localhost":
+        directory = "tests/functional_tests/accessibility_output"
+        if not os.path.exists(directory):
+            os.makedirs(directory, exist_ok=True)
+
+        axe = Axe()
+        results = axe.run(page)
+
+        wcag_violations = []
+
+        for violation in results['violations']:
+            if set(violation["tags"]) & set(ACCESSIBILITY_STANDARDS):
+                wcag_violations.append(violation)
+        
+        if len(wcag_violations) == 0:
+            assert "No WCAG accessibility issues found"
+        else:
+            # Cleans the URL to remove any invalid characters and replace with _
+            invalid_filename_chars = r'[\/:*?"<>|]'
+            sanitized_title = re.sub(invalid_filename_chars, '_', page.title())
+
+            max_title_len = 30
+            file_path = f"tests/functional_tests/accessibility_output/axe_results_{sanitized_title[:max_title_len]}.json"
```

Comment:
Should this be changed to use the directory set above, rather than defining the same base path twice?

E.g.
```
output_directory = "tests/functional_tests/accessibility_output"

file_name = f"axe_results_{sanitized_title[:max_title_len]}.json"

file_path = os.path.join(output_directory, file_name)
```

Reply:
Updated to use the above directory